docs/source/tutorial_notebooks/production_schema.ipynb

```diff
@@ -91,13 +93,20 @@
    },
    "outputs": [],
    "source": [
+    "# Production datasets can't be overwritten, so let's generate a random unique name\n",
```

Comment:
This is understandable in this context, but normally - and, I would think, especially for production datasets - the name should be meaningful. People may want to use it as a filter when querying.   I'd add a sentence making it clear that normally this is **not** how you would name a production dataset.